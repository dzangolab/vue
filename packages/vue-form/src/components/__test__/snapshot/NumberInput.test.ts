import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";

import NumberInput from "../../NumberInput.vue";

describe("NumberInput", () => {
  it("matches snapshot", () => {
    const wrapper = mount(NumberInput, {
      props: {
        label: "Number input",
        placeholder: "Placeholder",
      },
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
