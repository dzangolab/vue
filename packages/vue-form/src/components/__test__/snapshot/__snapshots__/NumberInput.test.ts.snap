// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NumberInput > matches snapshot 1`] = `
<div
  class="field number"
>
  <label
    for="number"
  >
    Number input
  </label>
  
  <input
    class=""
    id="input-field-number"
    name="number"
    placeholder="Placeholder"
    tabindex="0"
    type="number"
  />
  <!---->
  
</div>
`;
